src/main/java/gui/DiseaseAnalyzerGUI.java

```diff
@@ -30,54 +32,90 @@ public DiseaseAnalyzerGUI() throws Exception {
         this.setContentPane(DiseaseAnalyzerGUI);
         this.setResizable(true);
         this.setSize(10000,10000);
+        invalidInput.setVisible(false);
+        invalidInput.setText("invalid input please enter again");
+        returnToMenu.setText("Return To Menu");
```

Comment:
we can probably use constants for these messages to keep it neater and more consistent? this applies to all guis so we can just clean it up later

Reply:
Yep, the strings will be moved to the Constant in the final rounds of refactoring